### PR TITLE
Side menu

### PR DESCRIPTION
**Pull request by 21pranav**
Dummy Review

### CODE REVIEW

File: webapp/application.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ creativei_app.controller("MainController",function($scope, $rootScope, $state, $
       delete $scope.$storage.isAuthenticated;
       delete $rootScope.runningOrders;
       delete $scope.$storage.runningOrders;
-      return;
+       return;
```
Comment (aayushKhandpur) [inline webapp/application.js]: Remove extra space.

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Changes requested.

File: webapp/application.js (hunk 1 above)
Comment (21pranav) [reply webapp/application.js]: fixed

**Review by aayushKhandpur — APPROVED** [review]
Please merge

**Pull request merged**